bdsf/functions.py

```diff
@@ -1120,80 +1125,36 @@ def read_image_from_file(filename, img, indir, quiet=False):
 
     # Check that file exists
     if not os.path.exists(image_file):
-        img._reason = 'File does not exist'
+        img._reason = f'File {image_file} does not exist'
         return None
 
     # If img.use_io is set, then use appropriate io module
-    if img.use_io != '':
+    if img.use_io:
+        # Sanity check: only 'fits' and 'rap' are supported image I/O types
+        if img.use_io not in ('fits', 'rap'):
+            raise ValueError("Invalid image I/O type '{img.use_io}'. "
```

Comment:
I think we need an f-string here?

Reply:
Good catch. Otherwise the sanity check would become a source of error.